### ISSUE TITLE
Live Metrics should respect the DisableTelemetry switch

### ISSUE THREAD

**Issue by SergeyKanzhelev** [Microsoft/ApplicationInsights-dotnet-server#328]
As mentioned here: https://github.com/Microsoft/ApplicationInsights-aspnetcore/issues/348 live metrics should respect `DisableTelemetry` switch. All other modules should stop collecting data, not just disable it on channel. See https://github.com/Microsoft/ApplicationInsights-dotnet/issues/397

**Comment by matthieugd** [comment]
Is this repository also for the ASP.NET Core implementation? Because I didn't specify it in my original issue but we are using the nuget package 2.0 of the core implementation

**Comment by SergeyKanzhelev** [comment]
@matthieugd yes, we just reference the NuGet from this repository https://github.com/Microsoft/ApplicationInsights-aspnetcore/blob/develop/src/Microsoft.ApplicationInsights.AspNetCore/project.json#L97

**Comment by tokaplan** [comment]
What's the idea behind the kill switch? Is it to cut all communication with the service? Is it to save CPU cycles that would be otherwise consumed by telemetry collection? It's not immediately clear what a kill switch would mean for Live Metrics.

**Comment by matthieugd** [comment]
@tokaplan how do we disable live metrics? We disabled telemetry but if you look at the screenshot the CPU and Memory are still collected. It's confusing. If I don't need metrics at all I don't see why CPU and memory usage are still reported on the dashboard.

**Comment by tokaplan** [comment]
I understand the confusion; my question is - in your particular case, what's your motivation for disabling the telemetry in the first place?

**Comment by matthieugd** [comment]
Oh simply we don't want to pollute the data with metrics from development machines which are not representative of the system in "production"

**Comment by tokaplan** [comment]
Does the same apply to Live Metrics? You simply wouldn't want to see those dev boxes in Live Metrics on top of those "real" prod servers?

**Comment by matthieugd** [comment]
Ideally yes.

**Comment by Dmitry-Matveev** [comment]
@tokaplan , have you got any news on this item?
